lighthouse-core/runner.js

```diff
@@ -184,6 +143,47 @@ class Runner {
     }
   }
 
+  /**
+   * User can run -G solo, -A solo, or -GA together
+   * -G and -A will run partial lighthouse pipelines,
+   * and -GA will run everything plus save artifacts and lhr to disk.
+   *
+   * @template {LH.Config.Config | LH.Config.FRConfig} TConfig
+   * @param {(runnerData: {config: TConfig, driverMock?: Driver}) => Promise<LH.Artifacts>} gatherFn
+   * @param {{config: TConfig, driverMock?: Driver}} options
+   * @return {Promise<LH.Artifacts>}
+   */
+  static async gatherAndManageArtifacts(gatherFn, options) {
+    const settings = options.config.settings;
+
+    // Gather phase
+    // Either load saved artifacts off disk or from the browser
+    let artifacts;
+    if (settings.auditMode && !settings.gatherMode) {
+      // No browser required, just load the artifacts from disk.
+      const path = this._getDataSavePath(settings);
+      artifacts = assetSaver.loadArtifacts(path);
+      const requestedUrl = artifacts.URL.requestedUrl;
+
+      if (!requestedUrl) {
+        throw new Error('Cannot run audit mode on empty URL');
+      }
+    } else {
+      artifacts = await gatherFn({
+        config: options.config,
+        driverMock: options.driverMock,
+      });
+
+      // -G means save these to ./latest-run, etc.
```

Comment:
```suggestion
      // -G means save these to ./latest-run
```

(or: what does etc. refer to?)

Reply:
Idk I copied this. Probably refers to a user defined latest run folder.

Reply:
yes, either what `-G` is set to or defaults to `./latest-run/`:

https://github.com/GoogleChrome/lighthouse/blob/da73d4ae0455576a196b6bba776820f29fc2c4f1/lighthouse-core/runner.js#L470-L474